doc/filter_tutorial.md

```diff
@@ -0,0 +1,376 @@
+
+ ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial3.png)
+
+
+At the end of this document there is a filtering example, used by the MMA group to vet SGRB candidates. Once you plug in your filter, MongoDB Compass will break it down for you in different steps. This filter has two $project stages before the $match stage. You can play around with these to modify your filter.
+
+ ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial4.png)
+
+
+
+### 4. Saving candidates
+Once you are done with the filtering, you can
+<ol type="a">
+  <li>  ‘Export to language’ (recommended, as MongoDB Compass is not good for saving your filters) - button next to ‘Create new +’ </li>
+    <li>  Save your current filter – Save button next to ‘Create new +’ </li>
+    <li>  Export your results as a json or csv file – click on the export button next to ‘Run’ and then choose your preferred format. </li>
+  </ol>
+
+### 5. Exploring
+ Use your favorite language to explore your json or csv file.
+
+
+### Future work:
+Add ZTF alerts
+### Filter example
+```
+[{
+$project: {
+  _id: 1,
+  candid: 1,
+  objectId: 1,
+  'prv_candidates.jd': 1,
+  'prv_candidates.magpsf': 1,
+  'prv_candidates.fid': 1,
+  'prv_candidates.isdiffpos': 1,
+  'prv_candidates.diffmaglim': 1,
+  isdiffpos: '$candidate.isdiffpos',
+  m_now: '$candidate.magpsf',
+  m_app: '$candidate.magap',
+  t_now: '$candidate.jd',
+  fid_now: '$candidate.fid',
+  sgscore: '$candidate.sgscore1',
+  sgscore2: '$candidate.sgscore2',
+  sgscore3: '$candidate.sgscore3',
+  srmag: '$candidate.srmag1',
+  srmag2: '$candidate.srmag2',
+  srmag3: '$candidate.srmag3',
+  sgmag: '$candidate.sgmag1',
+  simag: '$candidate.simag1',
+  rbscore: '$candidate.rb',
+  drbscore: '$candidate.drb',
+  magnr: '$candidate.magnr',
+  distnr: '$candidate.distnr',
+  distpsnr1: '$candidate.distpsnr1',
+  distpsnr2: '$candidate.distpsnr2',
+  distpsnr3: '$candidate.distpsnr3',
+  scorr: '$candidate.scorr',
+  fwhm: '$candidate.fwhm',
+  elong: '$candidate.elong',
+  nbad: '$candidate.nbad',
+  chipsf: '$candidate.chipsf',
+  gal_lat: '$coordinates.b',
+  ssdistnr: '$candidate.ssdistnr',
+  ssmagnr: '$candidate.ssmagnr',
+  ssnamenr: '$candidate.ssnamenr',
+  t_start: '$candidate.jdstarthist',
+  age: {
+   $subtract: [
+	'$candidate.jd',
+	'$candidate.jdstarthist'
+   ]
+  },
+  psfminap: {
+   $subtract: [
+	'$candidate.magpsf',
+	'$candidate.magap'
+   ]
+  }
+}
+}, {
+$project: {
+  objectId: 1,
+  t_now: 1,
+  t_start: 1,
+  m_now: 1,
+  fid_now: 1,
+  sgscore: 1,
+  rbscore: 1,
+  drbscore: 1,
+  magnr: 1,
+  distnr: 1,
+  scorr: 1,
+  gal_lat: 1,
+  ssdistnr: 1,
+  ssnamenr: 1,
+  ssmagnr: 1,
+  age: 1,
+  nbad: 1,
+  fwhm: 1,
+  elong: 1,
+  psfminap: 1,
+  prv_candidates: 1,
+  positivesubtraction: {
+   $in: [
+	'$isdiffpos',
+	[
+ 	1,
+ 	'1',
+ 	't',
+ 	true
+	]
+   ]
```

Comment:
I would try to put some of this on single lines instead of the way it is shown now. It looks like it is pretty hard to read. I don't know if this is really meant for human consumption, but if so then increasing readability could be good. If the linter automatically does this to your code then there's not much you can do about it.

Reply:
I changed this too, hopefully looks better now